tests/test_material_files.py

```diff
@@ -2,11 +2,15 @@
 import pytest
 
 from htsohm.material_files import random_number_density
+from htsohm.db.__init__ import Structure
```

Comment:
The whole point of __init__.py is that it never is imported directly.

Should just be:
from htsohm.db import Structure